Adds PlayStoreStrings file to automate metadata upload

loremattei opened the pull request:
This PR is the first of a small serie aimed to automate the metadata string translation process and it adds the PlayStoreStrings.po that contains the metadata strings that we use in the PlayStore.

After the merge, we'll enable a cron job that pulls the strings from this file and uploads them to the GlotPress project, so that we can simplify the current download/modify/upload cycle.

Further planned steps are:
 - to have a script that updates the .po file fetching the data from a standard text one
 - to add  the screenshot promo strings in order to have them localized.

mzorz commented on WordPress/metadata/PlayStoreStrings.po:
```diff
@@ -0,0 +1,63 @@
+
+msgctxt "release_note_101"
+msgid ""
+"10.1:\n"
```
May I suggest that we add the 10.2 proposed release notes as well, just to begin with the latest we have:
```
10.2:
- New feature: Something there caught your eye? Read it any time with Save for later! Even when you're offline!
- Various bugfixes.
```

loremattei commented:
Yeah! You're right Mario! I updated the file with the 10.2 release notes and dropped the old 10.0 because I think we should be fine with having the current and next release data in this file.

I'd still like to have this merged into `develop` before 10.3 cut so that we can verify the integration with the GP script/job. We are going to reuse an existing script on the GP side, so we have to make sure that our file format conforms to the one required by it. And I'd like to have this checked and confirmed before starting to develop other scripts on top of it.
Does it make sense?

mzorz commented:
Thanks for the explanation!
Makes sense :)
:shipit:
